src/components/Header.astro

```diff
@@ -0,0 +1,28 @@
+---
+
+---
+
+<div class="grid grid-cols-1 md:grid-cols-2 justify-evenly items-center p-10 bg-dark-blue">
+    <div>
+        <div class="mb-6 md:mb-0">
+            <span class="self-center text-5xl font-semibold whitespace-nowrap text-white"><a href="https://lfx.linuxfoundation.org/tools/mentorship/" class="flex items-center text-center" target="_blank">LFX Mentorship</a></span>
+        </div>
+    </div>
+    <div class="relative">
+        <div class="absolute inset-y-0 left-0 flex items-center pl-3 pointer-events-none">
+            <svg class="w-4 h-4 text-gray-500 dark:text-gray-400" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 20 20">
+                <path stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="m19 19-4-4m0-7A7 7 0 1 1 1 8a7 7 0 0 1 14 0Z"/>
+            </svg>
+        </div>
+        <input type="search" id="default-search" class="block w-full p-4 pl-10 text-sm text-gray-900 border border-gray-300 rounded-lg bg-gray-50 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500" placeholder="Search Orgs..." required>
+        <button type="submit" class="text-white absolute right-2.5 bottom-2.5 font-medium rounded-lg text-sm px-4 py-2 bg-highlight-blue hover:bg-dark-blue duration-200">Search</button>
+    </div>
+</div>
+<div class="text-xl text-white bg-dark-blue mb-5 px-10 py-5">
+    <ul class="flex flex-col gap-2 sm:flex-row sm:justify-between">
+        <li><a href="https://google.com/" class="hover:text-highlight-blue duration-200">Home</a></li>
+        <li><a href="https://google.com/" class="hover:text-highlight-blue duration-200">Organisations</a></li>
```

Comment:
We will follow British English, thus `Organizations` instead of `Organisations.`